src/dappsExplorer/DAppsExplorerScreenFilter.tsx

```diff
@@ -88,6 +88,10 @@ export function DAppsExplorerScreenFilter() {
     horizontalScrollView.current?.scrollTo({ x: 0, animated: true })
   }
 
+  const filterPress = (filterId: string) => {
+    selectedFilter === filterId ? setSelectedFilter('all') : setSelectedFilter(filterId)
```

Comment:
do we need to ensure that `all` cannot be toggled? e.g. `selectedFilter === filterId || selectedFilter === 'all' ? setSelectedFilter('all') : setSelectedFilter(filterId)`

Reply:
I don't think we need to ensure 'alll cannot be toggled. When we tap all and it's already selected we shouldn't do anything special since the `selectedFilter === 'all' === filterId` and we call `setSelectedFilter('all')` and the button remains selected. 

With the example code I was unable to toggle other filters and playing around with it left me with the same results as what's already implemented.

Reply:
oh right, of course! makes sense